upp/configs/single-b.yaml

```diff
@@ -1,30 +1,30 @@
-# In this config, we don't have seperate samples for r2 or r3, we just combine them
 variables: !include variables.yaml
-#transform: !include transform.yaml
 
 ttbar: &ttbar
   name: ttbar
   equal_jets: False
   pattern:
-    - "mc20/user.nekumari.410470.e6337_s3681_r13144_p5770.tdd.EMPFlow.24_2_14.23-08-14_PFlow_Aug_23_output.h5/*.h5"
-    - "mc23/user.nekumari.601229.e8514_s4162_r14622_p5770.tdd.EMPFlow.24_2_14.23-08-14_PFlow_Aug_23_output.h5/*.h5"
-    - "mc23/user.nekumari.601230.e8514_s4162_r14622_p5770.tdd.EMPFlow.24_2_14.23-08-14_PFlow_Aug_23_output.h5/*.h5"
+    - "user.svanstro.410470.e6337_s3681_r13144_p5922.tdd.EMPFlow_kfold.24_2_27.23-11-10_kfoldtrain_output.h5/*.h5"
+    - "user.svanstro.601229.e8514_s4162_r14622_p5922.tdd.EMPFlow_kfold.24_2_27.23-11-10_kfoldtrain_output.h5/*.h5"
+    - "user.svanstro.601230.e8514_s4162_r14622_p5922.tdd.EMPFlow_kfold.24_2_27.23-11-10_kfoldtrain_output.h5/*.h5"
 
 zprime: &zprime
   name: zprime
   equal_jets: False
   pattern:
-    - "mc20/user.nekumari.800030.e7954_s3681_r13144_p5770.tdd.EMPFlow.24_2_14.23-08-14_PFlow_Aug_23_output.h5/*.h5"
-    - "mc20/user.nekumari.800030.e7954_s3797_r13144_p5770.tdd.EMPFlow.24_2_14.23-08-14_PFlow_Aug_23_output.h5/*.h5"
-    - "mc23/user.nekumari.800030.e8514_s4162_r14622_p5770.tdd.EMPFlow.24_2_14.23-08-21_PFlow_Aug_23_fix_output.h5/*.h5"
+    - "user.svanstro.800030.e7954_s3681_r13144_p5922.tdd.EMPFlow_kfold.24_2_27.23-11-10_kfoldtrain_output.h5/*.h5"
+    - "user.svanstro.800030.e7954_s3797_r13144_p5922.tdd.EMPFlow_kfold.24_2_27.23-11-19_kfoldtrain_output.h5/*.h5"
+    - "user.svanstro.800030.e8514_s4162_r14622_p5922.tdd.EMPFlow_kfold.24_2_27.23-11-10_kfoldtrain_output.h5/*.h5"
 
 global_cuts:
   train:
-    - [eventNumber, "%10<=", 7]
+    - [jetFoldHash, "%4!=", 0]
+    - [eventNumber, "%10<=", 8]
   val:
-    - [eventNumber, "%10==", 8]
-  test:
+    - [jetFoldHash, "%4!=", 0]
     - [eventNumber, "%10==", 9]
```

Comment:
If we wanted to make the training stats a little bit higher I suppose we could have split the holdout set into test and val, but this is fine too

Reply:
I also don't think the ~10% ish extra stats we get will have a significant effect on the models performance.

Reply:
good points!